locale/en.json

```
@@ -473,7 +471,7 @@
           },
           {
             "smallTitle": "Contact us for help",
-            "desc": "If you need help understanding your Home Office documents, you can call <span class='govuk-body govuk-!-font-weight-bold'> 0300 123 1711</span>or email <a class=\"govuk-link\" href=\"mailto:customer.service@justice.gov.uk\">customer.service@justice.gov.uk</a>"
+            "desc": "If you need help understanding your Home Office documents, you can call <span class='govuk-body govuk-!-font-weight-bold'> 0300 123 1711</span>or email <a class=\"govuk-link\" href=\"mailto:customer.service@justice.gov.uk\":customer.service@justice.gov.uk</a>"
```

I believe this is a typo
the colons should be after the word email and html syntax will break.
I'll add a suggestion:

```suggestion
            "desc": "If you need help understanding your Home Office documents, you can call <span class='govuk-body govuk-!-font-weight-bold'>0300 123 1711</span>or email: <a class=\"govuk-link\" href=\"mailto:customer.service@justice.gov.uk\">customer.service@justice.gov.uk</a>"
```